### DISCUSSION TITLE
how should persistance of the created project is happening in the Karavan?? .

### DISCUSSION THREAD

**Discussion by kondetiramanakumar** [apache/camel-karavan#1272] [Q&A] [answered]
### Ask a question

could you please  tell how  should be the  persistence of  the created project  is happening  in the Karavan Integration Tool ?????? any database is using??? or hazlecast IN Memory storage???..

Current Me If I am wrong. I checked the karavan open sources are using up the Hazlecast In Memory storage for storing temporarily the project files . But after stop and start using the Maven command : mvn clean compile quarkus:dev -Dquarkus.profile=local,public all my created project files are disappear. could you please throw some light on it Marat???

### Variant

Web Application

### Container Management (if applicable)

None

### Operating System (if applicable)

None

### Version

None

**Comment by kondetiramanakumar** [comment]
@mgubaidullin Glad to e-meet you Marat. Trust you are doing Great. Could you please throw some light on my above query???

**Comment by mgubaidullin** [comment, marked as answer]
ASF projects are volunteer based. Contributors spend their own personal time to make them happened.

Please, try to make a little bit of your own effort to find answers before asking questions.

<img width="994" alt="Screenshot 2024-05-22 at 9 04 27 AM" src="https://github.com/apache/camel-karavan/assets/1379213/133c79f1-237b-41db-af3d-e39a59aa043b">

**Comment by kondetiramanakumar** [comment]
@mgubaidullin Hello Marat, Trust things are going swimmingly at your end. Thanks a million for throwing some light on it.
